src/renderer_d3d12.h

```diff
@@ -20,8 +20,13 @@
 
 #if defined(__MINGW32__) // BK - temp workaround for MinGW until I nuke d3dx12 usage.
 extern "C++" {
+#if __cpp_constexpr >= 200704L && __cpp_inline_variables >= 201606L
+	__extension__ template<typename Ty>
+	constexpr const GUID& __mingw_uuidof();
```

Comment:
bgfx is currently compiling with C++14, why this wouldn't be compatible with C++14?

Reply:
The `#if` condition here is copied from the mingw64/x86_64-w64-mingw32/include/guiddef.h header.  The declaration only has the `constexpr` qualifier when compiling with C++17 features enabled.  Without the `#if` you’ll get a declaration mismatch one way or the other because the declaration you see from the MinGW headers will be different.